tests/fuzz/execution_context.cpp

```diff
@@ -118,6 +132,16 @@ TEST_CASE("execution_context_direct", "[fuzz]")
     ebpf_fuzzing_enabled = true;
     auto mt = seed_random_engine();
 
+    // Limit this processes memory to 50Mb
```

Comment:
```suggestion
    // Limit this processes memory to 50Mb.
```
But did you really mean "MB" (megabytes) instead of "Mb" (megabits)?  Line 139 suggests you meant "MB"